common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -25,6 +25,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gogo/protobuf/types"
+
+	"github.com/temporalio/temporal/common/persistence/sql"
```

Comment:
Isn't it weird?

Reply:
I mean cassandra reference to sql.

Reply:
Yes, this is necessary until we have datablob interfaces and I migrate the sql serializer into the 'PayloadSerializer' interface in which case it will move to managers level.